lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -6,31 +6,39 @@
  */
 @use '../../scss-shared/nx-focus-border-helpers';
 
-$-height: 16px;
-$-width: 16px;
+$-size: 16px;
+$-checked-border-width: 4px;
 
 .nx-radio__circle {
-  height: $-height;
-  width: $-width;
+  background-color: var(--nx-color-component-background);
+  border-radius: 50%;
+  height: $-size;
+  width: $-size;
 }
 
-.nx-radio__inner-circle {
-  fill: var(--nx-swatch-white);
-  stroke: var(--nx-swatch-blue-40);
-}
-
-.nx-radio__input:checked {
-  + .nx-radio__circle .nx-radio__outer-circle {
-    fill: none;
+.nx-radio__input {
+  &:disabled {
+    + .nx-radio__circle {
+      background-color: var(--nx-color-radio-disabled-background);
+      box-shadow:
+        inset 0 0 0 1px var(--nx-color-radio-disabled-border),
```

Comment:
Pretty sure this first shadow does nothing that isn't also accomplished by the second shadow